checkstyle/google_checks.xml

```diff
@@ -0,0 +1,327 @@
+<?xml version="1.0"?>
```

Comment:
Instead of adding another copy of this file `google_checks.xml` should be moved to `checkstooyle/google_checks.xml`

Reply:
I added the "original one" google_checks.xml verbatim as of a couple a days ago to know what the initial version was. Then the folio_checks.xml with the few changes. Yes. google_checks.xml is a copy but from a source that might change. What's the directory "checkstooyle" you are referring to?

Reply:
Typo.
Thank you for removing ./google_checks.xml in favour of ./checkstyle/google_checks.xml